pyk/src/pyk/kcfg.py

```diff
@@ -253,8 +257,42 @@ def resolve(node_id: str) -> str:
         for verified_ids in dct.get('verified') or []:
             cfg.add_verified(resolve(verified_ids['source']), resolve(verified_ids['target']))
 
+        for alias, id in dct.get('aliases', {}).items():
+            cfg.add_alias(name=alias, id=resolve(id))
+
         return cfg
 
+    def aliases(self, node_id: str) -> List[str]:
+        node_id = self._resolve(node_id)
+        return [alias for alias, value in self._aliases.items() if node_id == value]
+
+    def short_id(self, node: Node) -> str:
+        """ Return a user friendly name for a node. This may be either an alias or a shortened hash. """
+        for alias, hash in self._aliases.items():
+            if node.id == hash:
+                return '@' + alias
+        return shorten_hash(node.id)
```

Comment:
Do we need this function? I think displaying one alias without having an explicit default alias might be confusing,